canary/package.json

```diff
@@ -7,6 +7,9 @@
     "build": "yarn react && yarn vue && yarn angular",
     "pull": "chmod +x ./environments/pull-environments.sh && ./environments/pull-environments.sh",
     "install": "yarn react:install && yarn vue:install && yarn angular:install",
+    "script:install": "yarn --cwd script",
```

Comment:
Nit: alphabetize script order

Reply:
Handled in #1744 -- [287b3e7](https://github.com/aws-amplify/amplify-ui/pull/1744/commits/287b3e7fa3f37d6011b43e564cdad80dddcab219)